### PR TITLE
Add support for new September 14th 2021 devices.

### PR DESCRIPTION
**Pull request by Zandor300**
## Required
- [x] Device Identifiers

## Optional
- [ ] Support URLs
- [ ] Device images

### CODE REVIEW

**Comment by insightmind** [comment]
According to the device support files in Xcode 13 RC 
```
Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/DeviceTypes
```
these are the device identifiers:

| Device | Identifier |
| ------- | -------- |
| iPhone 13 mini | iPhone14,4 |
| iPhone 13 Pro Max | iPhone14,3 |
| iPhone 13 Pro | iPhone14,2 |
| iPhone 13 | iPhone14,5 |
| iPad mini (6th generation) | iPad14,1 |
| iPad (9th generation) | iPad12,2 |

Although not necessarily relevant for this PR the device identifiers for Apple Watch Series 7
```
Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/DeviceTypes
```

| Device | Identifier |
| ------- | -------- |
| Apple Watch Series 7 - 41mm | Watch6,6 |
| Apple Watch Series 7 - 45mm | Watch6,9 |

This is probably an incomplete list  of actually available device identifiers.

File: Source/Device.generated.swift [hunk 1]
```diff
@@ -417,6 +441,10 @@ public enum Device {
       case "iPhone13,1": return iPhone12Mini
       case "iPhone13,3": return iPhone12Pro
       case "iPhone13,4": return iPhone12ProMax
+      case "iPhoneTODO1": return iPhone13
+      case "iPhoneTODO2": return iPhone13Mini
+      case "iPhoneTODO3": return iPhone13Pro
```
Comment (earltedly) [inline Source/Device.generated.swift:446]: iPhone14,2 - [source](https://www.macrumors.com/2021/09/15/iphone-13-pro-gpu-performance-geekbench/)

**Comment by Zandor300** [comment]
@insightmind Thanks!

File: Source/Device.generated.swift [hunk 2]
```diff
@@ -680,33 +722,33 @@ public enum Device {
...
     /// All Pro Devices
     public static var allProDevices: [Device] {
-      return [.iPhone11Pro, .iPhone11ProMax, .iPhone12Pro, .iPhone12ProMax, .iPadPro9Inch, .iPadPro12Inch, .iPadPro12Inch2, .iPadPro10Inch, .iPadPro11Inch, .iPadPro12Inch3, .iPadPro11Inch2, .iPadPro12Inch4, .iPadPro11Inch3, .iPadPro12Inch5]
+      return [.iPhone11Pro, .iPhone11ProMax, .iPhone12Pro, .iPhone12ProMax, .iPhone13Pro, .iPhone13ProMax, .iPadPro9Inch, .iPadPro12Inch, .iPadPro12Inch2, .iPadPro10Inch, .iPadPro11Inch, .iPadPro12Inch3, .iPadPro11Inch2, .iPadPro12Inch4, .iPadPro11Inch3, .iPadPro12Inch5]
     }
 
     /// All mini Devices
     public static var allMiniDevices: [Device] {
-      return [.iPadMini, .iPadMini2, .iPadMini3, .iPadMini4, .iPadMini5]
+      return [.iPadMini, .iPadMini2, .iPadMini3, .iPadMini4, .iPadMini5, .iPadMini6]
```
[the remark this answers is not in the document]
Comment (denisenepraunig) [reply Source/Device.generated.swift:751]: Just wondering in the future if the iPhone Minis (12 and 13) are also mini devices. 🤔

**Review by denisenepraunig — APPROVED** [review]
Nice work - thank you @Zandor300 and @insightmind for the identifiers!

I will merge it with the device identifiers - the images and support articles take some time.

**Pull request merged**